python/lsst/pipe/tasks/hips.py

```diff
@@ -604,3 +614,907 @@ def build_quantum_graph(
             raise RuntimeError("Given constraints yielded empty quantum graph.")
 
         return pipeBase.QuantumGraph(quanta={task_def: quanta})
+
+
+class HipsPropertySpectralTerm(pexConfig.Config):
+    lambda_min = pexConfig.Field(
+        doc="Minimum wavelength (nm)",
+        dtype=float,
+    )
+    lambda_max = pexConfig.Field(
+        doc="Maximum wavelength (nm)",
+        dtype=float,
+    )
+
+
+class HipsPropertyConfig(pexConfig.Config):
+    """Configuration parameters for writing a HiPS property file."""
+    creator_did_template = pexConfig.Field(
+        doc=("Unique identifier of the HiPS - Format: IVOID. "
+             "Use ``{band}`` to substitute the band name."),
+        dtype=str,
+        optional=False,
+    )
+    obs_collection = pexConfig.Field(
+        doc="Short name of original data set - Format: one word",
+        dtype=str,
+        optional=True,
+    )
+    obs_description_template = pexConfig.Field(
+        doc=("Data set description - Format: free text, longer free text "
+             "description of the dataset.  Use ``{band}`` to substitute "
+             "the band name."),
+        dtype=str,
+    )
+    prov_progenitor = pexConfig.ListField(
+        doc="Provenance of the original data - Format: free text",
+        dtype=str,
+        default=[],
+    )
+    obs_title_template = pexConfig.Field(
+        doc=("Data set title format: free text, but should be short. "
+             "Use ``{band}`` to substitute the band name."),
+        dtype=str,
+        optional=False,
+    )
+    spectral_ranges = pexConfig.ConfigDictField(
+        doc=("Mapping from band to lambda_min, lamba_max (nm).  May be approximate."),
+        keytype=str,
+        itemtype=HipsPropertySpectralTerm,
+        default={},
+    )
+    initial_ra = pexConfig.Field(
+        doc="Initial RA (deg).  If not set will use a point in MOC.",
```

Comment:
Is this "initial" as in a hint to the position at which a HiPS viewer should center the display?